pkg/generator/rootca.go

```diff
@@ -38,7 +38,7 @@ func (rCA *RootCA) References() ([]string, []string) {
 }
 
 // LoadReferenceData loads references from data
-func (rCA *RootCA) LoadReferenceData(data []map[string][]byte) error {
+func (rCA *RootCA) LoadReferenceData(data map[string][]byte) error {
```

Comment:
This would have to be updated in all keyTypes. I think this should happen as part of this PR

Reply:
@jrcast :+1: I need to update the type you merged. 

- [ ] update password type
- [ ] update literal type